desktop/cmp/persistenceManager/PersistenceManager.ts

```diff
@@ -15,67 +15,67 @@ import {pluralize} from '@xh/hoist/utils/js';
 interface PersistenceManagerProps extends HoistProps<PersistenceManagerModel> {
     /** True to disable options for saving/managing items. */
     minimal?: boolean;
+    /** True (default) to render a save button alongside the primary menu button when dirty. */
+    enableTopLevelSaveButton?: boolean;
+    /** True to omit the default menu component. Should be used when creating custom app-specific component */
+    omitDefaultMenuComponent?: boolean;
 }
 
 export const [PersistenceManager, persistenceManager] =
     hoistCmp.withFactory<PersistenceManagerProps>({
         displayName: 'PersistenceManager',
         model: uses(PersistenceManagerModel),
 
-        render({model, minimal = false}) {
-            const {
-                    selectedView,
-                    isShared,
-                    entity,
-                    manageDialogModel,
-                    saveDialogModel,
-                    omitDefaultMenuComponent
-                } = model,
-                displayName = entity.displayName;
-
-            return fragment(
-                hbox({
-                    className: 'xh-persistence-manager',
-                    items: [
-                        popover({
-                            omit: omitDefaultMenuComponent,
-                            item: button({
-                                text: model.getHierarchyDisplayName(selectedView?.name) ?? `-`,
-                                icon: isShared ? Icon.users() : Icon.bookmark(),
-                                rightIcon: Icon.chevronDown(),
-                                outlined: true
-                            }),
-                            content: div({
-                                items: [
-                                    div({
-                                        className: 'xh-popup__title',
-                                        item: capitalize(pluralize(displayName))
-                                    }),
-                                    objMenu({minimal})
-                                ]
-                            }),
-                            placement: 'bottom-start'
-                        }),
-                        saveButton()
-                    ]
-                }),
-                manageDialog({omit: !manageDialogModel}),
-                saveDialog({omit: !saveDialogModel})
-            );
+        render({model, ...props}) {
+            return fragment(defaultMenu({...props}), manageDialog(), saveDialog());
```

Comment:
To discuss - either exporting the dialogs, exposing more props for customizing, or making them pluggable (i.e. `PersistenceManagerProps` could accept a `manageDialogCmp` and `saveDialogCmp` that default to our Hoist implementations)